### PR TITLE
letter done 62.jpg

### PR DESCRIPTION
**Pull request by Bhosle07**

### CODE REVIEW

**Review by laxmikant-gaykhee — CHANGES_REQUESTED** [review]
-Add **style.css** in **css** folder 
-Add **images** in **img** folder
-Add **preview.html** 
-**index.html** in this file give only **background div**

**Review by laxmikant-gaykhee — APPROVED** [review]

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]
css folder name must be in lower case

File: SubmittedAssignments/Bhosle_Mahesh_letter/.vscode/settings.json
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_letter/.vscode/settings.json:1]: remove this folder

File: SubmittedAssignments/Bhosle_Mahesh_letter/CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+.topbox1{
+    height: 8mm;
+    width: 100%;
+    background: rgb(88, 191, 101);
+}
+
+.topbox2{
+    height: 13mm;
+    width: 81mm;
+    border: 3px solid rgb(88, 191, 101);
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_letter/CSS/style.css:29]: do not use px

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

**Pull request closed without merging**